### PR TITLE
Add new special world property ignorefirestate

### PR DESCRIPTION
**Pull request by FileEX**
This PR adds a new ~~glitch~~ special world property, ``ignorefirestate`` which allows getting into burning vehicles (with fire particle, not the fire indicating low HP and explosion), as well as aiming 'special' weapons like the ``camera``, ``sniper``, ``RPG``, and ``rocket launcher HS`` while the ped is on fire. Thanks to @ds1-e  for the idea.

### CODE REVIEW

**Comment by TheNormalnij** [comment]
It is not a sa glitch. This feature has more sense in `setWorldSpecialPropertyEnabled`

**Review by Fernando-A-Rocha — CHANGES_REQUESTED** [review]
Good job!

I agree with TheNormalnij, this feature should be added in a world property as you said it's not a proper glitch, but a different behavior like burn flipped cars.

**Comment by FileEX** [comment]
Moved to special world property

**Review by Fernando-A-Rocha — APPROVED** [review]

File: Client/mods/deathmatch/logic/CClientGame.cpp
Comment (bum8hj) [inline Client/mods/deathmatch/logic/CClientGame.cpp:1]: A little nitpick on an unrelated change:
I would rather write "Kickout ..." as 2 words, like "Kick out ..."

**Pull request merged**